modular_skyrat/modules/death_consequences_perk/death_consequences_trauma.dm

```diff
@@ -441,16 +451,12 @@
 		message += span_danger("\nWhile dead, subject will suffer degradation at a rate of [span_bolddanger("[base_degradation_reduction_per_second_while_alive] per second")].")
 		if (owner_organic && formaldehyde_death_degradation_mult != 1)
 			message += span_danger(" In such an event, formaldehyde will alter the degradation by <b>[span_blue("[formaldehyde_death_degradation_mult]")]</b>x.")
-		if (on_stasis_death_degradation_mult < 1)
+		if (on_stasis_passive_degradation_mult < 1)
 			message += span_danger(" Stasis may be effective in slowing (or even stopping) degradation.")
 	if (base_degradation_on_death)
 		message += span_danger("\nDeath will incur a <b>[base_degradation_on_death]</b> degradation penalty.")
 	if (owner_organic && rezadone_degradation_decrease)
-		message += span_danger("\nRezadone of purity at or above <i>[rezadone_minimum_purity]</i>% will reduce degradation by [span_blue("[rezadone_degradation_decrease]")] per second when metabolized.")
-	if (owner_organic && strange_reagent_degradation_decrease)
-		message += span_danger("\nStrange reagent will reduce degradation by [span_blue("[strange_reagent_degradation_decrease]")] per second when metabolized.")
-	if (owner_organic && sansufentanyl_degradation_decrease)
-		message += span_danger("\nSansufentanyl will reduce degradation by [span_blue("[sansufentanyl_degradation_decrease]")] per second when metabolized.")
+		message += span_danger("\nRezadone of purity at or above <i>[DEATH_CONSEQUENCES_REZADONE_MINIMUM_PURITY]</i>% will reduce degradation by [span_blue("[rezadone_degradation_decrease]")] per second when metabolized.")
 	if (eigenstasium_degradation_decrease)
 		message += span_danger("\nEigenstasium will reduce degradation by [span_blue("[eigenstasium_degradation_decrease]")] per second when present.")
```

Comment:
Do you actually *have* to do the `"[eigenstasium_degradation_decrease]"` here, or would `eigenstasium_degradation_decrease` just work? I'm curious, I can't remember if it works right in DM or not.

Reply:
you have to stringify it